dpctl/tests/test_sycl_platform.py

```diff
@@ -212,3 +213,49 @@ def test_get_platforms():
             assert has_sycl_platforms()
     except Exception:
         pytest.fail("Encountered an exception inside get_platforms().")
+
+
+def test_platform_get_devices():
+    platforms = dpctl.get_platforms()
+    if platforms:
+        for p in platforms:
+            assert len(p.get_devices())
+    else:
+        pytest.skip("No platforms available")
```

Comment:
It seems there is no need for skip here, we can just exit silently.

Reply:
We have generally preferred to skip in such cases, simply to give the user the message. I think I prefer it this way as well.